### ISSUE TITLE
Weight link die 

### ISSUE THREAD

**Issue by trand2k** [ruizengalways/PFNet#7]
Hi author,
Your weight link of model is die, can you give me weight file of this model ?
thank you

**Comment by makiisthenes** [comment]
Yes I am having the same issue, is it possible for you to update link
